### PR TITLE
update pre-commit linters

### PR DESCRIPTION
**Pull request by zaro0508**
* Nothing depends on managed-ec2-v5.yaml template anymore so we remove it.
* Update pre-commit linters
* Fix linter error..
```
AWS CloudFormation Linter................................................Failed
- hook id: cfn-python-lint
- exit code: 4

W2030 You must specify a valid allowed value for InstanceType (g2.8xlarge).
Valid values are ["a1.2xlarge", "a1.4xlarge", .... "x2iezn.8xlarge"... "z1d.metal", "z1d.xlarge"]
templates/managed-ec2-v5.yaml:75:9 
```

### CODE REVIEW

File: templates/managed-ec2-v5.yaml [hunk 1]
```diff
@@ -72,7 +72,6 @@ Parameters:
       - c5.9xlarge
       - c5.18xlarge
       - g2.2xlarge
-      - g2.8xlarge
```
Comment (brucehoff) [inline templates/managed-ec2-v5.yaml:75]: Just curious:  Why remove g2.8xlarge instances?

**Review by brucehoff — COMMENTED** [review]
Just curious: Why remove g2.8xlarge instances?

**Review by xschildw — APPROVED** [review]
+1 on @brucehoff 's question

File: templates/managed-ec2-v5.yaml (hunk 1 above)
Comment (zaro0508) [reply templates/managed-ec2-v5.yaml:75]: good catch,  i just noticed that nothing depends on `managed-ec2-v5.yaml` template anymore so I just removed it.

**Pull request merged**